frontend/src/pages/documentation/Contribute.js

```diff
@@ -69,32 +71,28 @@ const Contribute = ({ history, ...props }) => {
       title: `Publish your Operator on OperatorHub.io`,
       content: (
         <React.Fragment>
-          <p>OperatorHub.io is front-end for the Community Operator Repository: </p>
-          <p>
-            <ExternalLink href={operatorsRepo} text={operatorsRepo} indicator={false} />
-          </p>
           <p>
+            OperatorHub.io is the front-end for the{' '}
+            <ExternalLink href={operatorsRepo} text={"Community Operators Repository"} indicator={false} />
             This is an open source community project aiming to curate and collect Kubernetes Operators. Community
-            contributions will live in:
+            contributions live in:
           </p>
           <p>
             <ExternalLink href={contributions} text={contributions} indicator={false} />
```

Comment:
Right now we direct users to contribute to `upstream-community-operators` here, and again on [this line](https://github.com/operator-framework/operatorhub.io/pull/107/files#diff-76c00fe9908322921c4c17ee3c1cb0c6R223). It might be prudent to discuss the difference between `community-operators` and `upstream-community-operators` here.